app/gui2/src/components/GraphEditor/widgets/WidgetText.vue

```diff
@@ -36,13 +59,29 @@ export const widgetDefinition = defineWidget(WidgetInput.isAstOrPlaceholder, {
 </script>
 
 <template>
-  <!-- See comment in GraphNode next to dragPointer definition about stopping pointerdown -->
-  <EnsoTextInputWidget v-model="value" class="WidgetText r-24" @pointerdown.stop />
+  <label class="WidgetText r-24" @pointerdown.stop>
+    <NodeWidget :input="WidgetInput.FromAst(inputTextLiteral?.open ?? emptyTextLiteral.open!)" />
+    <AutoSizedInput v-model.lazy="textContents" />
+    <NodeWidget :input="WidgetInput.FromAst(inputTextLiteral?.close ?? emptyTextLiteral.close!)" />
```

Comment:
I think this could display mismatched open/close tokens in some cases, like:

`a = "unclosed`

The text literal implicitly closes at EOL--I think this PR would display the text as `"unclosed'` until it is edited.

```
a = '''
    multiline
```

The opener is `'''`, and there is no closer; I think this would display it as `'''multiline'` (there are probably other issues with multiline text though--I don't think that's been addressed specifically yet).

Reply:
My intention was to preserve the the quotes that are used in the code for as long as possible. When you edit an uninterpolated text, as long as the new contents don't contain illegal character (the quote itself), it will stay as uniterpolated.

I've now implemented your suggestion, and in case of omitted close token, the open token is displayed twice.

I'm still going to mostly ignore handling of multi-line blocks. Once we know how we want to deal with both blocks and multi-line strings in nodes in general, we can revisit what needs to change here. I believe the current implementation will strip all newline characters on edit, since this is the behavior of html input fields.